### ISSUE TITLE
Add command to import .po files

### ISSUE THREAD

**Issue by gabooh** [drush-ops/drush#2164]
This repository (https://github.com/dawehner/drush_language) is what I'm looking for. Just used, works fine.
I think it would make sense to move this to drush core as it's a common need.

I know `drush locale-update` can be used but it's way more complicated than just `drush language-import fr yourfile.po`. See https://www.drupal.org/node/2715369 for a comparison of what's needed.

**Comment by tstoeckler** [comment]
Well `drush locale-update` is really useful if you maintain your PO files in version control. In that case you don't want have to explicitly import each file every time, but just run `drush locale-check`and `drush locale-update` once. I agree, though, that explicitly importing and exporting PO files would be a useful addition. In fact there's even a `@todo` to that effect in `drush.locale.inc`

**Comment by TessaBakker** [comment]
See also https://www.drupal.org/project/drush_locale_sync

**Comment by Sutharsan** [comment]
I want to deploy custom translations (translations that are set in the admin interface to override community translations). I want to deploy them by exporting and importing custom translations. None of the available drush modules provide this option. And therefore I have written my own.

But because I believe in collaboration, and I think that drush core should at least provide the options that Drupal core offers, I propose the following import function:

```
Imports a single gettext file.

Examples:
 locale-import nl drupal-8.4.2.nl.po
 locale-import nl custom-translations.po --type=custom --override=all  

Arguments:
 language		The language code of the imported translation file.
 file			Path and file name of the gettext file.

Options:
 --type			The type of translations to be imported. Defaults to non-custom.
 			Options:
 			- custom: Treat imported strings as custom translations.
			- non-custom: Treat imported strings as non-custom translations.
 --override		Whether and how imported strings will override existing translations. 
 			Defaults to the Import behavior configurred in the admin 
			interface. Options:
			- none: Don't overwrite existing translations. Only append new
			  translations.
			- custom: Only override existing customized translations.
			- non-custom: Only override non-customized translations,
			  customized translations are kept.
			- all: Override any existing translation.
```

**Comment by Sutharsan** [comment]
The `locale:import` command became available in Drush 9.5.0. See PR  #3521 for details.
I guess this issue can be closed now.
